httpserver/build.gradle.kts

```diff
@@ -24,13 +24,14 @@ application {
 }
 
 dependencies {
+  api("de.fxlae:typeid-java-jdk8:0.2.0")
+  api("xyz.block:web5:0.0.9-delta")
+
   implementation(project(":protocol"))
   implementation(project(":httpclient"))
-  implementation("de.fxlae:typeid-java-jdk8:0.2.0")
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.14.2")
   implementation("com.fasterxml.jackson.core:jackson-databind:2.15.2")
   implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.16.0")
-  implementation("com.github.TBD54566975:web5-kt:v0.0.9-gamma")
```

Comment:
this is the line which (I think? can't recreate locally) is actually causing [the build issue](https://jitpack.io/com/github/TBD54566975/tbdex-kt/0.7.1-beta/build.log)